src/styles/index.scss

```diff
@@ -173,6 +167,36 @@ font-size: 20px;
 	letter-spacing: 0.2px;
 	font-size: 17px;
 }
-.view-btn:hover{
-	background: #097205;;
+.view-btn:hover {
+	background: #097205;
+}
+
+// Small devices (landscape phones, 576px and up)
+@media (min-width: 576px) {
+}
+
+// Medium devices (tablets, 768px and up)
+@media (min-width: 768px) {
+	.search-part {
```

Comment:
I'd put these `@media` queries **inside** of the element they're being applied to:
```scss
.search-form {
  foo: bar;
  @media (min-width: 768px) {
    foo: baz;
  }
}

Reply:
i did this ...thanks